modules/adyoulikeBidAdapter.md

```diff
@@ -47,13 +51,17 @@ Banner formats are supported.
           "required": true,
           "sizes": []
         }
+      },
+      video: {
+        playerSize: [640, 480],
+        context: 'instream'
       }
-      bids: [{
-        bidder: "adyoulike",
-        params: {
-          placement: 194 f787b85c829fb8822cdaf1ae64435,
-          DC: "fra01", // Optional for set the data center name
-        }
-      }]
-    };
+    },
+    bids: [{
+      bidder: "adyoulike",
+      params: {
+        placement: 194f787b85c829fb8822cdaf1ae64435
```

Comment:
wrap this code in quotes (`"`). Also, is this a test code? I can't see any test ad with this param.

Reply:
I've update the placement id so you can use it with our test tool.
please go to https://fo-debug.omnitagjs.com/debug/force
Fill the campaign id  with  c7f8f3d9d66f483048b2bc40df3b231c  and submit form.
This will set a cookie to force a test video campaign response.